analyzers/.sonarlint/SonarAnalyzer.slconfig

```diff
@@ -9,7 +9,7 @@
   "Profiles": {
     "CSharp": {
       "ProfileKey": "AWT63_RpV-oLu4FdMlyI",
-      "ProfileTimestamp": "2020-12-28T10:06:24Z"
+      "ProfileTimestamp": "2021-03-26T13:27:24Z"
```

Comment:
@csaba-sagi-sonarsource I now realize that these are auto-generated files by SonarLint and maybe we should just ignore the SLVS folder.

WDYT?

Reply:
Indeed, I agree with you.

Reply:
the only problem is that it's referenced in each project... : `    <AdditionalFiles Include="..\..\.sonarlint\sonaranalyzer-dotnet\CSharp\SonarLint.xml" Link="Properties\SonarLint.xml" />`